app/views/layouts/blacklight.html.erb

```diff
@@ -28,8 +28,8 @@
 
     <%# Only track analytics if configured. Preproduction tracks to separate GA account for testing. %>
 
-    <% if ENV['TRACK_GOOGLE_ANALYTICS'] == 'true' && defined?(ENV['GOOGLE_ANALYTICS_TRACKING_ID']) %>
-      <%= render 'google_analytics' %>
+    <% if defined?(ENV['GOOGLE_ANALYTICS_TRACKING_ID']) %>
```

Comment:
defined? is always going to return "method" whether or not GOOGLE_ANALYTICS_EMBED_ID is set. Use present? method instead.

`<% if ENV['GOOGLE_ANALYTICS_EMBED_ID'].present? %>`